src/components/trade/PriceImpact/utils.ts

```diff
@@ -0,0 +1,107 @@
+import BigNumber from 'bignumber.js'
+
+import { parseBigNumber } from 'utils'
+import { ONE_HUNDRED_BIG_NUMBER, ONE_BIG_NUMBER, ZERO_BIG_NUMBER } from 'const'
+
+import { PriceImpactArgs, PriceImpactArgsBase } from '../PriceImpact'
+
+// Limit as to where price colour changes
+const HIGH_PRICE_IMPACT_THRESHOLD = new BigNumber(5)
+const MID_PRICE_IMPACT_THRESHOLD = new BigNumber(3)
+const LOW_PRICE_IMPACT_THRESHOLD = ONE_BIG_NUMBER
+// TODO: remove for slippage toggle
+// 0.5% placeholder slippage
+const PLACEHOLDER_SLIPPAGE = new BigNumber(0.005)
```

Comment:
The deep in the market percentage should be bigger imo. 
Maybe ask rafa, but at least 5%?

0.5% or 1% could be for the lighter version of the warning

Reply:
I think i handled this in the new logic so will resolve, take a look at fixes please